config.py.sample

```diff
@@ -37,6 +37,9 @@ MUSICBRAINZ_CLIENT_SECRET = "CLIENT_SECRET"
 LASTFM_API_KEY = "USE_LASTFM_API_KEY"
 
 
+# Max time in seconds after which the playing_now stream will expire.
+PLAYING_NOW_MAX_DURATION = 5 * 60
```

Comment:
Ah, I see your point. 

Yes, the removing the last now_playing happens when the next one comes in, the key is overwritten. But you make a good point about what happens when a user stops listening. 80 mins is a bit far out. I would say we should keep it simple for now. I'd say go with 10 minutes, rather than 5. Do I hear a 7.5? :)